### PR TITLE
Update contract

### LINKED ISSUE #10: [Blockchain] Add vote ciphertext to blockchain contract
- [ ] Authenticate token sent with vote
- [x] Define appropriate contract for votes

### PR DESCRIPTION
**Pull request by Fantaskink**
Added a route to get the current election phase at ``election/current-phase``. Added ``objectId``to candidate and party in smart contract. Updated routes and controllers to reflect changes.

### CODE REVIEW

**Review by Jatewo — DISMISSED** [review]
LGTM 👍

File: controllers/candidate.js [hunk 1]
```diff
@@ -17,13 +17,13 @@ async function getCandidates(req, res, next) {
...
-async function publishCandidate(name, party) {
+async function publishCandidate(objectId, name, party) {
     const election = getElection();
     if (election === null) {
         throw new Error('Election has not started');
     }
 
-    if (!name || !party) {
+    if (!objectId || !name || !party) {
         throw new Error('Candidate name and party are required');
```
Comment (Mafusn) [inline controllers/candidate.js]: 'Candidate object ID, name and party are required' instead of just name and party

**Review by Mafusn — DISMISSED** [review]
Only one very small comment

File: controllers/candidate.js (hunk 1 above)
Comment (Fantaskink) [reply controllers/candidate.js]: Fixed in be58054

**Review by Mafusn — APPROVED** [review]
Very nice 💯

**Review by Jatewo — APPROVED** [review]

**Pull request merged**